src/Test/Utilities/Portable/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -634,6 +641,11 @@ private static bool CanBeInControlFlowGraph(IOperation n)
                 case OperationKind.IsNull:
                 case OperationKind.CaughtException:
                 case OperationKind.StaticLocalInitializationSemaphore:
+                case OperationKind.Discard:
+                case OperationKind.TupleBinaryOperator:
+                case OperationKind.ObjectOrCollectionInitializer: // PROTOTYPE(dataflow): it looks like this node is leaking through in some error scenarios, at least for now.
```

Comment:
This isn't leaking though in error scenarios, it's intended behavior that this will be in the resulting control flow graph, with flow references replacing initializer elements as required.

Reply:
Or nevermind, I'm just misremembering things.

---
In reply to: [188744224](https://github.com/dotnet/roslyn/pull/26903#discussion_r188744224) [](ancestors = 188744224)

Reply:
> This isn't leaking though in error scenarios, it's intended behavior that this will be in the resulting control flow graph, with flow references replacing initializer elements as required.

I saw this for error cases specifically without IObjectCreationOperation node in the original tree. There are references to specific scenarios in implementation file. In any case, this is just a temporary comment and it doesn't have to be 100% accurate.

---
In reply to: [188744224](https://github.com/dotnet/roslyn/pull/26903#discussion_r188744224) [](ancestors = 188744224)